setup.py

```diff
@@ -37,6 +37,12 @@
 # Always include standard site-packages to PYTHONPATH
 os.environ['PYTHONPATH'] = sysconfig.get_paths()['purelib']
 
+# Produce static Rust binaries.
+#
+# This is required to use the same virtme-ng-init across different root
+# filesystems (when `--root DIR` is used).
+os.environ["RUSTFLAGS"] = os.environ.get("RUSTFLAGS", "") + " -C target-feature=+crt-static"
```

Comment:
I guess the user will not want to override this new default flag, so no need to put the user's flag at the end, right? :)

Reply:
Good point! (Without the space before `-C` :) )

I don't know what's better. I guess it is not common to export `RUSTFLAGS` in the `.bashrc` file or similar, and use `virtme-ng` with `--root`.

Reply:
Yeah, just to avoid potential breakage let's go with the safest way: add our static flag at the beginning and append the user's RUSTFLAGS at the end (pushed another updated).